### ISSUE TITLE
A11Y: The WAI-ARIA role(s) button are not valid for the element svg.

### ISSUE THREAD

**Issue by bjliuyi** [carbon-design-system/carbon-components-react#523]
## Detailed description
Followed with accessibility checkpoint, the use of ARIA in HTML should meet the document conformance requirement for the element. But, TooltipSimple and Tooltip failed in this point.

Checkpoint: https://www.w3.org/TR/html-aria/#docconformance

Describe in detail the issue you're having. Is this a feature request (new component, new icon), a bug, or a general issue?
It is a general issue.

> Is this issue related to a specific component?
No

> What did you expect to happen? What happened instead? What would you like to see changed?
Role = img

> What browser are you working in?
Chrome

> What version of the Carbon Design System are you using?
"carbon-components-react": "5.0.6",

## Additional information
* Screenshots or code
![image](https://user-images.githubusercontent.com/34873342/35210291-9aa12eac-ff8c-11e7-9258-74cb9400677d.png)

## Add labels
Role="img" in element svt.
